src/plugins/data/public/search/search_interceptor.ts

```diff
@@ -52,6 +53,7 @@ export interface SearchInterceptorDeps {
   startServices: Promise<[CoreStart, any, unknown]>;
   toasts: ToastsSetup;
   usageCollector?: SearchUsageCollector;
+  uiActions: UiActionsStart;
```

Comment:
is uiactions not available in core start? since this one is the default interceptor did we want to consider making part of the query enhancements plugin's extension of this interceptor

Reply:
How would I add it as an extension? I could make it an optional field

Reply:
Made the dep optional